src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala

```diff
@@ -85,11 +86,12 @@ sealed trait App extends SbtReactiveAppKeys {
 
     endpoints := {
       val endpointName = akkaClusterBootstrapEndpointName.value
+      val managementEndpointName = akkaClusterBootstrapManagementEndpointName.value
       val bootstrapEnabled = enableAkkaClusterBootstrap.value.getOrElse(akkaClusterBootstrapEnabled.value)
 
       endpoints.?.value.getOrElse(Seq.empty) ++ {
         if (bootstrapEnabled)
-          Seq(TcpEndpoint(endpointName, 0))
+          Seq(TcpEndpoint(endpointName, 0), TcpEndpoint(managementEndpointName, 0))
```

Comment:
Probably we should add `def apply(name: String)` that defaults the port to `0` to reactive-lib, some other time I think